fhir-server/src/test/java/au/csiro/pathling/fhirpath/parser/ParserTest.java

```diff
@@ -836,15 +836,7 @@ void testQuantityAdditionWithOverflow() {
         .selectResult()
         .hasRows(spark, "responses/ParserTest/testQuantityAdditionWithOverflow_code.csv");
   }
-
-  @Test
-  void testTraversalToUnsupportedReferenceChild() {
-    final String expression = "reverseResolve(MedicationRequest.subject).requester.identifier";
-    final InvalidUserInputError error = assertThrows(InvalidUserInputError.class,
-        expression);
-    assertEquals("No such child: " + expression, error.getMessage());
-  }
-
+  
```

Comment:
I think we should replace this test with one that verifies that we can traverse to one of the newly encoded fields within `Reference`.

Reply:
Yes, added in the new test format.